buzzmobile/simulation/models/car/car_joints.urdf

```diff
@@ -342,15 +356,21 @@
     <joint name="lidar_joint" type="fixed">
         <parent link="chassis"/>
         <child link="laser"/>
-        <origin xyz="0.255 0 0.110" rpy="0 0 0"/>
+        <origin xyz="0.255 0 5" rpy="0 0 0"/>
```

Comment:
so all you did here was put the lidar way above the car. can you actually place it where our lidar is IRL?